priv/templates/extended_bin

```diff
@@ -129,25 +107,35 @@ if [ -z "$RELX_CONFIG_PATH" ]; then
     fi
 fi
 
+orig_vmargs_path="$VMARGS_PATH.orig"
+if [ $RELX_REPLACE_OS_VARS ]; then
+    # if there is no vm.args.orig then make a copy of the
+    # the original vm.args
+    if [ ! -f "$orig_vmargs_path" ]; then
+       mv "$VMARGS_PATH" "$orig_vmargs_path"
+    fi
+
+    # apply the environment variable substitution to vm.args.orig
+    # the result is saved to vm.args
+    awk '{while(match($0,"[$]{[^}]*}")) {var=substr($0,RSTART+2,RLENGTH -3);gsub("[$]{"var"}",ENVIRON[var])}}1' < "$orig_vmargs_path" > "$VMARGS_PATH"
+fi
+
```

Comment:
What will happen if you run the second time without RELX_REPLACE_OS_VARS? Would you want to run it from modified vm.args or original vm.args?